packages/sirius-web/frontend/sirius-web-application/src/navigationBar/NavigationBar.tsx

```diff
@@ -38,19 +39,22 @@ const useNavigationBarStyles = makeStyles()((theme) => ({
   },
   toolbar: {
     display: 'grid',
-    gridTemplateColumns: '1fr min-content 1fr',
+    gridTemplateColumns: '1fr 1fr 1fr',
```

Comment:
Why change that? It creates a massive space for the center area

Reply:
Done.